lib/command/src/Obelisk/Command/Run.hs

```diff
@@ -70,37 +74,67 @@ data CabalPackageInfo = CabalPackageInfo
 preprocessorIdentifier :: String
 preprocessorIdentifier = "__preprocessor-apply-packages"
 
-run :: MonadObelisk m => m ()
+-- | Imports needed to use Obelisk.Run
+obRunImports :: [String]
+obRunImports = [ "import qualified Obelisk.Run"
+               , "import qualified Frontend"
+               , "import qualified Backend" ]
+
+profile
+  :: MonadObelisk m
+  => FilePath
+  -> m ()
+profile profileBaseName = withProjectRoot "." $ \root -> do
+  freePort <- getFreePort
+  assets <- findProjectAssets root
+  putLog Debug $ "Assets impurely loaded from: " <> assets
+  putLog Debug "Using profiled build of project."
+  liftIO $ createDirectoryIfMissing False $ takeDirectory profileBaseName
+  let -- Sane flags to enable by default, enable time profiling +
+      -- closure heap profiling.
+      rtsFlags = [ "+RTS", "-p", "-po" <> profileBaseName, "-hc", "-RTS" ]
+      nixBuildExpr = unlines $
```

Comment:
We actually already depend on `here` in this project. You can use the `[iTrim|...|]` quasiquoter to make this much prettier.

Reply:
thanks! that makes things much cleaner.